### PR TITLE
Polishing statistics (EXPOSUREAPP-4693)

### PR DESCRIPTION
**Pull request by jurajkusnier**
- Unify statistics cards design

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -145,6 +145,6 @@
     <!-- Contact Diary -->
     <dimen name="contact_diary_list_item">64dp</dimen>
     <dimen name="contact_diary_nested_recyclerview_margin">60dp</dimen>
-    <dimen name="statistics_card_height">328dp</dimen>
+    <dimen name="statistics_card_height">352dp</dimen>
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values/dimens.xml:148]: IMHO this only mitigates the problem, one pixel phones you are quite limited regarding the font size. I'll test it on a Samsung device.

Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/values/dimens.xml:148]: still working on a better layout fix

**Comment by harambasicluka** [comment]
Galaxy 8 with an strange font, biggest available font size and in large:
image

standard font with largest possible size:
image

File: Corona-Warn-App/src/main/res/values/dimens.xml (hunk 1 above)
Comment (LukasLechnerDev) [inline Corona-Warn-App/src/main/res/values/dimens.xml:148]: This is differs to the figma designs and uses too much whitespace when using regular font size. 

As discussed on Teams we should revert it and probably implement a dynamic card height until Friday.

**Review by LukasLechnerDev — CHANGES_REQUESTED** [review]
Please revert card height, otherwise LGTM.

**Review by LukasLechnerDev — APPROVED** [review]

File: Corona-Warn-App/src/main/res/layout/home_statistics_cards_incidence_layout.xml [hunk 2]
```diff
@@ -22,28 +22,19 @@
...
-            android:layout_height="wrap_content"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
-
         <TextView
             android:id="@+id/value_label"
             style="@style/StatisticsCardValueLabel"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginTop="16dp"
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/layout/home_statistics_cards_incidence_layout.xml:38]: I think @d4rken just changed that to be fixed

**Review by chiljamgossow — COMMENTED** [review]
I believe you reverted some recent changes made for increased font size

**Comment by jurajkusnier** [comment]
> I believe you reverted some recent changes made for increased font size

Yes, that's possible, I adjusted some changes to have the cards look the same. Font size adaptation will be part of another PR anyway.

**Pull request merged**